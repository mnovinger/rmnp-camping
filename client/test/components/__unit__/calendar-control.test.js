import renderer from 'react-test-renderer';
import CalendarControl from '../../../src/app/components/calendar-control';
import React from 'react';

describe('The CalendarControl Component', () => {
  it('should render correctly', () => {
    const props = {
      incrementWeekOffset: () => {},
      decrementWeekOffset: () => {}
    }

    const tree = renderer.create(< CalendarControl { ...props }/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

});