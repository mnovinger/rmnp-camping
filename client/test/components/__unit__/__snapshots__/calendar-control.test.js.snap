// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The CalendarControl Component should render correctly 1`] = `
<div
  className="calendar-control"
>
  <img
    className="cal-back-arrow cal-arrows"
    onClick={[Function]}
    src="kuba_arrow_back.png"
  />
  <div
    className="cal-control-text"
  >
    Move calendar
  </div>
  <img
    className="cal-forward-arrow cal-arrows"
    onClick={[Function]}
    src="kuba_arrow_forward.png"
  />
</div>
`;
